docs/tutorials/hello-world.md

```diff
@@ -180,9 +175,7 @@ println!(
 
 ## Custom Database and Blockchain types
 
-We briefly mentioned before that for our example we used the `MemoryDatabase`, but that it could also be swapped for a different one: this is one example of the *modularity* of BDK. By default, some database
-types are implemented in the library, namely (as of now) the `MemoryDatabase` which only keeps data in RAM and the [sled][sled] database that can store data on a filesystem. But since the `Database` trait is public,
-users of the library can also implement different database types more suitable for their use-case.
+We briefly mentioned before that for our example we used the `MemoryDatabase`, but that it could also be swapped for a different one: this is one example of the *modularity* of BDK. By default, some database types are implemented in the library, namely (as of now) the `MemoryDatabase` which only keeps data in RAM and the [sled][sled] database that can store data on a filesystem. But since the `Database` trait is public, users of the library can also implement different database types more suitable for their use-case.
```

Comment:
Add `[rusqlite]: https://docs.rs/rusqlite/latest/rusqlite/` after line 193.


```suggestion
We briefly mentioned before that for our example we used the `MemoryDatabase`, but that it could also be swapped for a different one: this is one example of the *modularity* of BDK. By default, some database types are already implemented in the library, namely the `MemoryDatabase` which only keeps data in RAM, the [sled][sled] database that can store data on a filesystem, and the `SqliteDatabase`[rusqlite] that can store data into a SQLite database. But since the `Database` trait is public, users of the library can also implement different database types more suitable for their use-case.
```